src/components/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+import { Loader } from '../Loader';
+import { PeopleTable } from '../Loader/PeopleTable';
+import { getPeople } from '../services/api';
+import { Person } from '../../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getPeople()
+      .then(data => {
+        const peopleByName: { [key: string]: Person } = {};
+
+        data.forEach(person => (peopleByName[person.name] = person));
+        const dataUpdated = data.map(p => {
+          return {
+            ...p,
+            mother: peopleByName[p.motherName || ''],
+            father: peopleByName[p.fatherName || ''],
+          };
+        });
+
+        setPeople(dataUpdated);
```

Comment:
This is a code quality issue. It's better to handle errors in a more detailed way. Instead of simply setting the error state to true, consider also storing the error message and displaying it to the user. This way, you can give the user more context about what went wrong.